KS Auto Service/gallery.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="./img/favicon-icon-24.jpg" type="image/x-icon" />
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/general.css">
+  <link rel="stylesheet" href="./css/header.css">
+  <link rel="stylesheet" href="./css/style.css">
+  <link rel="stylesheet" href="./css/gallery.css">
+  <link rel="stylesheet" href="./css/dark-theme.css">
+  <link rel="stylesheet" href="./css/light-theme.css">
+  <title>KS Auto Service Gallery</title>
+</head>
+
+<body class="dark-theme">
+  <div class="container">
+    <div class="main-block">
+      <div class="head">
+        <header>
+          <div class="work-hours">
+            <p>Mon - Fri: 08:00 - 18:00</p>
+            <p>Sut - Sun: Day off</p>
+          </div>
+
+          <img class="img-logo" src="img/logo.png" alt="Logo">
+
+          <div class="work-adress">
+            <p>Podskalská 2054/8 Prague 128 00</p>
+          </div>
+        </header>
+
+        <nav class="nav-menu">
+          <a href="index.html">Home</a>
+          <a href="gallery.html">Gallery</a>
+          <a href="contact-us.html">Contact Us</a>
+        </nav>
+      </div>
+    </div>
+    <h1 class="h1-gallery">Gallery</h1>
+    <main>
+      <img src="img/gallery/1.jpg" alt="Photo">
+      <img src="img/gallery/2.jpg" alt="Photo">
+      <img src="img/gallery/3.jpg" alt="Photo">
+      <img src="img/gallery/4.jpg" alt="Photo">
+      <img src="img/gallery/5.jpg" alt="Photo">
+      <img src="img/gallery/6.jpg" alt="Photo">
```

Comment:
alt attribute has to have some specific info what this photo is about
not just general text 'Photo'